core/src/com/unciv/logic/civilization/CivInfoStats.kt

```diff
@@ -125,10 +125,10 @@ class CivInfoStats(val civInfo: CivilizationInfo){
             }
         }
 
-        if (civInfo.containsBuildingUnique("Provides 1 happiness per social policy")) {
+        if (civInfo.containsBuildingUnique("Provides 1 happiness per 2 additional social policies adopted")) {
             if(!statMap.containsKey("Policies")) statMap["Policies"]=0f
             statMap["Policies"] = statMap["Policies"]!! +
-                    civInfo.policies.getAdoptedPolicies().count { !it.endsWith("Complete") }.toFloat()
+                    civInfo.policies.getAdoptedPolicies().count { !it.endsWith("Complete") } / 2f
```

Comment:
As I have already mentioned before, this change for `getAdoptedPolicies().count` = 5 will provide 2.5 happiness, instead of 2.0 as it says "1 happiness per 2 additional social policies".